config/pom.xml

```diff
@@ -11,90 +11,30 @@
   <name>Hono Configuration</name>
   <description>Default Hono configuration to be used by other modules.</description>
 
+  <build>
+    <resources>
+      <resource>
+        <directory>demo-certs/certs</directory>
+        <!--<filtering>true</filtering>-->
+      </resource>
+    </resources>
+    <!--<plugins>
```

Comment:
Here I have a doubt because we have two solutions. Due to the presence of binary files (p12 and jks), with the initial configuration which had filtering=true I had problems with corrupted binary files by the maven resources plugin. One way is to set filtering=false and the other one is leaving filtering=true but adding the non filtered file extensions (p12 and jks). What do you think ?